src/components/Input/Input.stories.tsx

```diff
@@ -1,60 +1,152 @@
 import * as React from "react";
 
 import Input from "./Input";
+import { InputTypes } from "./InputTypes";
+import Label from "../Label/Label";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+import { text, boolean, select } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
 
 export default {
     title: "Input",
     component: Input,
 };
 
+let showLabel, showHelperText;
+
 export const input = () => (
-    <Input
-        labelId="blah"
-        labelContent={<>Label</>}
-        inputId="blah"
-        isRequired={false}
-    ></Input>
+    <>
+        {boolean("Show Label", true) ? (showLabel = true) : (showLabel = false)}
+        {showLabel && (
+            <Label
+                htmlFor="inputID"
+                optReqFlag={select(
```

Comment:
Changing this knob doesn't change anything visually right now

Reply:
It does, but it's obscured by the "Show Info" storybook button... every time I try to work on storybook-docs it's a nonstarter since it's more complicated than just dropping in it, but damn that ugly blue button really cramps my vibes